activejob/CHANGELOG.md

```diff
@@ -1,3 +1,57 @@
+*   Allow `assert_performed_with` to be called without a block.
+
+    *bogdanvlviv*
+
+*   Execution of `assert_performed_jobs`, and `assert_no_performed_jobs`
+    without a block should respect passed `:except`, `:only`, and `:queue` options.
+
+    *bogdanvlviv*
+
+*   Allow `:queue` option to `assert_no_performed_jobs`. If the `:queue` option is specified,
+    then only the job(s) enqueued to a specific queue will not be performed.
+
+    Example:
+    ```
+    def test_assert_no_performed_jobs_with_queue_option
+      assert_no_performed_jobs queue: :some_queue do
+        HelloJob.set(queue: :other_queue).perform_later("jeremy")
+      end
+    end
+    ```
+
+    *bogdanvlviv*
+
+*   Allow `:queue` option to `assert_performed_jobs`. If the `:queue` option is specified,
+    then only the job(s) enqueued to a specific queue will be performed.
+
+    Example:
+    ```
+    def test_assert_performed_jobs_with_queue_option
+      assert_performed_jobs 1, queue: :some_queue do
+        HelloJob.set(queue: :some_queue).perform_later("jeremy")
+        HelloJob.set(queue: :other_queue).perform_later("bogdan")
+      end
+    end
+    ```
+
+    *bogdanvlviv*
+
+*   Allow `:queue` option to `perform_enqueued_jobs`. If the `:queue` option is specified,
+    then only the job(s) enqueued to a specific queue will be performed.
+
+    Example:
+    ```
+    def test_perform_enqueued_jobs_with_queue
+      perform_enqueued_jobs queue: :some_queue do
+        MyJob.set(queue: :some_queue).perform_later(1, 2, 3) # will be performed
+        HelloJob.set(queue: :other_queue).perform_later(1, 2, 3) # will not be performed
+      end
+      assert_performed_jobs 1
+    end
+    ```
+
+    *bogdanvlviv*
+
```

Comment:
Let's combine into one changelog entry that heralds these together, that `:queue` may be used with job assertions & helpers.

Reply:
Done in https://github.com/rails/rails/commit/c6f4ddf34531f5bc017eb7e5cde0fabd5dd411cc.